yxi/axi-calyx/axi-combined-calyx.futil

```diff
@@ -95,9 +95,8 @@ component m_arread_channel(
       // this all needs to be one group
       // this contains blocking logic previously in its own group
       group do_ar_transfer {
-          //assert ARVALID
-
-          is_arvalid.in = !(is_arvalid.out & ARREADY) & !arvalid_was_high.out ? 1'b1;
+          //assert ARVALID as long as this is the first time we are asserting it
+          is_arvalid.in = !arvalid_was_high.out ? 1'b1;
```

Comment:
It seems to me like `is_arvalid.out & ARREADY` will always be true because `is_arvalid.out` is always 0 as we call `deassert_val` in the control sequence:

```
while {
  // some stuff ...
  do_ar_transfer;
  deassert_val;
  // more stuff ...
}

So we can simplify to just `!arvalid_was_high.out` as our guard